src/lib/Input/Input.types.ts

```diff
@@ -31,7 +31,7 @@ export type InputArgs = {
 export type CustomArgs = {
   isMulti?: boolean;
   selectOptions: SelectOption[];
-  selectValue?: string | number;
+  selectValue?: string | number | MultiValue<SelectOption>;
```

Comment:
Oh nice, ignore my comment in the other CMA PR regarding this, didnt realise you had a PR here to solve it